<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QDialog" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>279</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/qp.ico</normaloff>res/qp.ico</iconset>
  </property>
  <widget class="QPushButton" name="m_pedsButton1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="gui.qrc">
     <normaloff>:/res/BTN_UP.png</normaloff>:/res/BTN_UP.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_quitButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MPool[0]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_epoolLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_crossing">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>498</width>
     <height>220</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gui.qrc">:/res/crossing.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="m_carsLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>32</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gui.qrc">:/res/cars_red.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="m_pedsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gui.qrc">:/res/peds_blank.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="m_onOffButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_stateLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>state</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pedsButton2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="gui.qrc">
     <normaloff>:/res/BTN_UP.png</normaloff>:/res/BTN_UP.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <zorder>m_crossing</zorder>
  <zorder>m_pedsButton1</zorder>
  <zorder>m_quitButton</zorder>
  <zorder>label</zorder>
  <zorder>m_epoolLabel</zorder>
  <zorder>m_carsLabel</zorder>
  <zorder>m_pedsLabel</zorder>
  <zorder>m_onOffButton</zorder>
  <zorder>m_stateLabel</zorder>
  <zorder>m_pedsButton2</zorder>
 </widget>
 <tabstops>
  <tabstop>m_pedsButton1</tabstop>
  <tabstop>m_quitButton</tabstop>
 </tabstops>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_quitButton</sender>
   <signal>clicked()</signal>
   <receiver>Gui</receiver>
   <slot>onQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pedsButton1</sender>
   <signal>pressed()</signal>
   <receiver>Gui</receiver>
   <slot>onPedsPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Gui</sender>
   <signal>finished(int)</signal>
   <receiver>Gui</receiver>
   <slot>onQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_onOffButton</sender>
   <signal>pressed()</signal>
   <receiver>Gui</receiver>
   <slot>onOnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_onOffButton</sender>
   <signal>released()</signal>
   <receiver>Gui</receiver>
   <slot>onOnReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pedsButton2</sender>
   <signal>pressed()</signal>
   <receiver>Gui</receiver>
   <slot>onPedsPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pedsButton1</sender>
   <signal>released()</signal>
   <receiver>Gui</receiver>
   <slot>onPedsReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pedsButton2</sender>
   <signal>released()</signal>
   <receiver>Gui</receiver>
   <slot>onPedsReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onQuit()</slot>
  <slot>onOnPressed()</slot>
  <slot>onOnReleased()</slot>
  <slot>onPedsPressed()</slot>
  <slot>onPedsReleased()</slot>
 </slots>
</ui>
